cmd/kube-controller-manager/app/plugins.go

```diff
@@ -43,7 +43,22 @@ func ProbeRecyclableVolumePlugins() []volume.VolumePlugin {
 	// The list of plugins to probe is decided by the kubelet binary, not
 	// by dynamic linking or other "magic".  Plugins will be analyzed and
 	// initialized later.
-	allPlugins = append(allPlugins, host_path.ProbeVolumePlugins()...)
-	allPlugins = append(allPlugins, nfs.ProbeVolumePlugins()...)
+	allPlugins = append(allPlugins, host_path.ProbeVolumePlugins(defaultRecycler(60))...)
```

Comment:
I agree the volume plugin and its recycler is mostly orthogonal.  I think that is expressed by having the Recycler interface and struct/impl separate from the current plugin's Builders and Cleaners.  The fact is, Recyclers (and by extension the Creators and Deleters we'll need for the rest of dynamic provisioning) don't strictly need to be part of the volume plugin.  

Whatever patterns we put in place for configuring and running Recyclers will be copied to the rest of the dyna-prov implementation, which I intended to start fairly soon after the dust settles from 1.0.

Is there an analog I can follow elsewhere that would give me insight into how configuring Recyclers should look?